backend/books/serializers.py

```diff
@@ -14,10 +14,11 @@ class BookListAddSerializer(serializers.ModelSerializer):
     """
     authors = serializers.SlugRelatedField(queryset=Author.objects.all(), many=True, slug_field='name')
     genres = serializers.SlugRelatedField(queryset=Genre.objects.all(), many=True, slug_field='name')
+    urls = serializers.StringRelatedField(many=True)
```

Comment:
Why are we doing many urls here? Would there not be a single url to a photo?

Reply:
yeah this is to account for the case where multiple images might be needed to represent a book.